config/fission.json

```diff
@@ -0,0 +1,16 @@
+{
+  "experiments": {
+    "update_code": false,
+    "update_storage": false,
+    "download_results": false,
+    "deployment": "fission",
+    "runtime": {
+      "language": "python",
+      "version": "3.6"
```

Comment:
We should update Python and Node versions :-)

Reply:
Yes I did that